app/src/main/java/org/projectbuendia/client/ui/chart/PatientChartActivity.java

```diff
@@ -420,30 +386,6 @@ private String getFormattedPcrString(double pcrValue) {
         }
 
         @Override public void updatePatientConditionUi(String generalConditionUuid) {
```

Comment:
Not necessarily bad -- after all the code is always retrievable from git.  Just didn't want the knowledge to disappear with the code.